core/lib/pubs.ts

```diff
@@ -85,3 +85,15 @@ export const processedPubToPubResult = <T extends InputPub>(pub: T): GetPubsResu
 export const processedPubsToPubsResult = (pubs: InputPub[]): GetPubsResult => {
 	return pubs.map(processedPubToPubResult);
 };
+
+export const getTitleField = <T extends InputPub>(
+	pub: T
+): T["pubType"]["fields"][number] | undefined => pub.pubType.fields.find((field) => field.isTitle);
+
+export const valuesWithoutTitle = <T extends InputPub>(pub: T): T["values"] => {
+	const titleField = getTitleField(pub);
+	if (!titleField) {
+		return pub.values;
+	}
+	return pub.values.filter((value) => value.fieldId !== titleField?.id);
+};
```

Comment:
I modified how we filter the title out of the values rendered on the pub page because it was relying on the title field being named `"Title"` which led to unintuitive behavior if there was a field named `"Title"` that wasn't the type's title, or if another field was the title.